conf/unbound.conf

```diff
@@ -0,0 +1,68 @@
+server:
```

Comment:
How different is this from the stock configuration file when ubound is installed?

Reply:
A number of the options here are indeed the default for a stock configuration. The main differences are probably in the performance and hardening settings. 
Some testing seems to confirm the unbound configuration will work out of the box.